server/pages/index.js

```diff
@@ -369,8 +371,10 @@ module.exports = function() {
 				const pathRegex = utils.pathToRegExp( path );
 
 				app.get( pathRegex, function( req, res, next ) {
+					req.context = Object.assign( {}, req.context, { sectionName: section.name } );
+
 					if ( config.isEnabled( 'code-splitting' ) ) {
-						req.context = Object.assign( {}, req.context, { chunk: section.name } );
+						req.context.chunk = section.name;
```

Comment:
@ehg Code cleanup. Since we're now using `Object.assign` above, and thus ensuring that `req.context` is an object, I think we're safe here just setting `req.context.chunk = ...` rather than using another `Object.assign`.